java/my-widget-service/bin/.gitignore

```diff
@@ -0,0 +1,13 @@
+.classpath.txt
+target
+.classpath
+.project
+.idea
+.settings
+.vscode
+*.iml
+
+# CDK asset staging directory
+.cdk.staging
+cdk.out
+
```

Comment:
add the java .class to your gitignore. they're not usually included in Code Reviews, right?

Reply:
ya i have added the bin directory to .gitignore.